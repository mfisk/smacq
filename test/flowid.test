#!/bin/sh

${BINDIR}smacqq -m -O <<EOF 
print flowid, srcip, dstip, packets, packetsback, start, finish from (flowid -r, srcip, dstip from pcapfile($TEST/0325@1112-snort.pcap))
EOF
