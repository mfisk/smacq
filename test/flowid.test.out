0	192.168.0.54	192.168.0.210	1	0	2000-03-25T12:13:29.360668-07:00
1	192.168.0.54	192.168.0.200	1	0	2000-03-25T12:15:28.020237-07:00
1	192.168.0.54	192.168.0.200	103	1	2000-03-25T12:15:28.020237-07:00	2000-03-25T12:15:53.354234-07:00
0	192.168.0.54	192.168.0.210	4	0	2000-03-25T12:13:29.360668-07:00	2000-03-25T12:13:29.363329-07:00
